migrations/versions/08894082221a_replace_collection_name_with_observer_.py

```
@@ -55,16 +55,28 @@ def upgrade() -> None:
+                    f"""
+                    WITH batch AS (
+                        SELECT id
+                        FROM {schema}.documents
+                        WHERE session_name IS NULL
+                        LIMIT :batch_size
+                    )
+                    UPDATE {schema}.documents
+                    SET session_name = '__global_observations__'
+                    FROM batch
+                    WHERE documents.id = batch.id
+                    """
```

Ruff (0.14.0)

Possible SQL injection vector through string-based query construction

(S608)